setup/quickstart/faq.md

```diff
@@ -9,6 +9,25 @@ sidebar:
 
 Here are a few Q/A pairs that come up fairly frequently.
 
+## I'm unable to load the Applications screen
+
+After installing Spinnaker and navigating to the <em>Applications</em> screen, you may see one of
+following issues:
+ * The loading indicator spins continuously (prior to release 1.9)
+ * The following error message is displayed: <em>Error fetching applications</em> (release 1.9 and later)
+
+ The most common cause of this error is that your browser cannot communicate with your Gate endpoint.
+ (This endpoint defaults to `http://localhost:8084`, but can be customized.)
+
+ Check your browser console log and/or network for any failed requests to `<gate-endpoint>/applications`.
+ Some things to check while troubleshooting:
+ * If you are accessing Spinnaker via the default `http://localhost:9000`, check that you have
+   forwarded Gate's port (8084 by default) to the machine where your browser is running.
+ * If you are accessing Spinnaker via a custom URL, ensure that you have set `override-base-url`
+   for both the UI (Deck) and API (Gate) services, as described in the question below.  Failure to do
+   this will lead to your browser rejecting requests to Gate due to their failing the CORS preflight
```

Comment:
What's the CORS preflight check? Worth mentioning? Or omitting here?

Reply:
It probably wasn't necessary for me to mention 'preflight' check at all as that's more of an implementation detail of how browsers work.  I've reworded this so that it's a bit more clear and less jargony.